assets/app/views/browse/build-config.html

```diff
@@ -22,7 +22,7 @@
                 <button
                     class="btn btn-default hidden-xs"
                     ng-click="startBuild()"
-                    ng-disabled="!canBuild">
+                    ng-disabled="!canBuild()">
```

Comment:
Twice for me on page load.

![openshift_web_console_and_builds_js____go_src_github_com_openshift_origin_assets_app_scripts_services__-_vim1](https://cloud.githubusercontent.com/assets/1167259/13333746/8a7301e4-dbd6-11e5-88de-e7657e00eb87.png)